pymilvus/milvus_client/milvus_client.py

```diff
@@ -0,0 +1,801 @@
+"""MilvusClient for dealing with simple workflows."""
+import logging
+from pprint import pprint
+import threading
+from typing import Optional, Union, List, Dict
+from uuid import uuid4
+
+from tqdm import tqdm
+
+from pymilvus.client.types import LoadState
+from pymilvus.exceptions import DataNotMatchException, MilvusException
+from pymilvus.milvus_client.defaults import DEFAULT_SEARCH_PARAMS
+from pymilvus.orm import utility
+from pymilvus.orm.collection import Collection, CollectionSchema, FieldSchema
+from pymilvus.orm.connections import connections
+from pymilvus.orm.types import DataType, infer_dtype_bydata
+
+logger = logging.getLogger()
+logger.setLevel(logging.DEBUG)
+
+
+class MilvusClient:
```

Comment:
I think you will need a add a close method to:
close the connection
cleanup all the collections, index and release the data if necessary.